README.md

```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
   ./gatk-launch PrintReadsSpark -I input.bam -O output.bam
   ```
 
+####Running GATK4 with inputs on Google Cloud Storage:
+
+* GATK can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
+  ```
+  ./gatk-launch PrintReads -I gs://mybucket/path/to/my.bam -L 1:10000-20000 -O output.bam
+  ```
+* For public datasets, that will just work. For private datasets you must set up authentication.
```

Comment:
Is this true?  Who gets charged for the data transfer in this case?  The owner of the data?